campari/simulation.py

```diff
@@ -252,31 +252,33 @@ def simulate_wcs(angle=None, x_shift=None, y_shift=None, roman_path=None, base_s
     """
     rotation_matrix = np.array([np.cos(angle), -np.sin(angle), np.sin(angle),
                                np.cos(angle)]).reshape(2, 2)
-    image = fits.open(roman_path + f"/images/simple_model/{band}/" +
-                      f"{base_pointing}/Roman_TDS_simple_model_{band}_{base_pointing}"
-                      + f"_{base_sca}.fits.gz")
+
+    img_collection = ImageCollection()
+    img_collection = img_collection.get_collection("ou2024")
+    image = img_collection.get_image(pointing=base_pointing, sca=base_sca, band=band)
+    header = image.get_fits_header()
 
     CD_matrix = np.zeros((2, 2))
-    CD_matrix[0, 0] = image[0].header["CD1_1"]
-    CD_matrix[0, 1] = image[0].header["CD1_2"]
-    CD_matrix[1, 0] = image[0].header["CD2_1"]
-    CD_matrix[1, 1] = image[0].header["CD2_2"]
+    CD_matrix[0, 0] = header["CD1_1"]
+    CD_matrix[0, 1] = header["CD1_2"]
+    CD_matrix[1, 0] = header["CD2_1"]
+    CD_matrix[1, 1] = header["CD2_2"]
 
     CD_matrix_rotated = CD_matrix @ rotation_matrix
 
     wcs_dict = {
-            "CTYPE1": image[0].header["CTYPE1"],
-            "CTYPE2": image[0].header["CTYPE2"],
-            "CRPIX1": image[0].header["CRPIX1"],
-            "CRPIX2": image[0].header["CRPIX2"],
+            "CTYPE1": header["CTYPE1"],
+            "CTYPE2": header["CTYPE2"],
+            "CRPIX1": header["CRPIX1"],
+            "CRPIX2": header["CRPIX2"],
             "CD1_1": CD_matrix_rotated[0, 0],
             "CD1_2": CD_matrix_rotated[0, 1],
             "CD2_1": CD_matrix_rotated[1, 0],
             "CD2_2": CD_matrix_rotated[1, 1],
-            "CUNIT1": image[0].header["CUNIT1"],
-            "CUNIT2": image[0].header["CUNIT2"],
-            "CRVAL1":   image[0].header["CRVAL1"] + x_shift,
-            "CRVAL2":  image[0].header["CRVAL2"] + y_shift,
+            "CUNIT1": header["CUNIT1"],
+            "CUNIT2": header["CUNIT2"],
+            "CRVAL1":   header["CRVAL1"] + x_shift,
+            "CRVAL2":  header["CRVAL2"] + y_shift,
```

Comment:
Remove extra spaces after `:`  Should just be 1 space.

(I know this was in the original, but while you're touching these lines, fix the little things.)